shap/plots/_text.py

```diff
@@ -894,22 +1084,44 @@ def get_color(shap_value,cmax):
                     }}
                     else {{
                         if (getIdSide_{uuid}({uuid}_heatmap_flat_state) === getIdSide_{uuid}(id)) {{
-                            document.getElementById({uuid}_heatmap_flat_state).style.backgroundColor  = "white";
-                            cleanValuesAndColors_{uuid}(id)
+                        
+                            // Another token clicked on, on the side side on the previously selected token
```

Comment:
fix comment, not sure what it should be:
                             // Another token clicked on, on the side side on the previously selected token

Reply:
Changed comment to be more clear about what is happening in condition